sdk/keyvault/keyvault-certificates/src/identifier.ts

```diff
@@ -39,26 +29,14 @@ export interface ParsedKeyVaultCertificatesIdentifier {
 }
 
 /**
- * Parses a KeyVaultIdentifier from a string URI.
+ * Parses the string Id from a Key Vault Certificate into the KeyVaultCertificateId type.
```

Comment:
Avoid providing information on types which can be inferred from the function signature. Consider using an example input so that it re-enforces that the input is a url like id. In .Net, they don't need to do this as the input type is uri.

```suggestion
 * Parses the given Key Vault Certificate Id of the form (add an example here) into different components like `name`, `vaultUrl` and `version`.
```

Reply:
Taking your feedback a bit further, I will be doing this instead:

```ts
/**
 * Parses the given Key Vault Certificate Id.
 * Key Vault Ids come in the shape of URIs, for example
 * a KeyVaultKey will have an `id` property, with a value similar to the following one:
 * 
 *   https://<keyvault-name>.vault.azure.net/keys/<key-name>/<unique-version-id>
 * 
 * This URI can be parsed into segments using this function. The previous URI would result in:
 * 
 *   {
 *      sourceId: "https://<keyvault-name>.vault.azure.net/keys/<key-name>/<unique-version-id>",
 *      vaultUrl: "https://<keyvault-name>.vault.azure.net",
 *      version: "<unique-version-id>",
 *      name: "<key-name>"
 *   }
 * 
 * @param {string} id The Id of the Key Vault Key.
 */
export function parseKeyVaultKeyId(id: string): KeyVaultKeyId {
/* ... */
```

Let me know what you think.

Reply:
Looks good, made a suggestion to shorten it a bit